internal/pkg/flink/internal/controller/.snapshots/TestInteractiveOutputControllerTestSuite-TestTableTitleDisplaysPageSizeAndCacheSizeWithUnsafeTrace

```diff
@@ -1 +1 @@
- Table mode (auto refresh 1.0s) | last page size: 10 | current cache size: 10/10 | table size: 10 
+ Table mode (auto refresh 1.0s) | last page size: 10 | current cache size: 10/10 | table size: 10 | pending retraction: 0 
```

Comment:
Nit: Any reason the first column is capitalized, but the others aren't?

Reply:
Also, out of curiosity what does "pending retraction" mean?

Reply:
Not really. Retraction is a flink specific term telling us “this row is no longer valid, they may be a replacement coming soon”

Reply:
First column is the only one you see outside of debug mode